### PR TITLE
Added regisgtration tunnel between browser-facade and auth-service through main-service

### PR DESCRIPTION
**Pull request by jakublaz**
It is working now small code changes maybe need something more returning from auth-service?

### CODE REVIEW

File: main-service/src/server.js
Comment (piotr-ladkowski) [inline main-service/src/server.js:1]: Please refrain from editing the files which do not belong th the scope of the issue, since it may create a lot of conflicts with others working on different files

File: docker-compose.yml
Comment (piotr-ladkowski) [inline docker-compose.yml:1]: what is the reason for this change?

**Comment by piotr-ladkowski** [comment]
@jakublaz please answer the questions and later i will approve

File: docker-compose.yml
Comment (jakublaz) [reply docker-compose.yml:1]: so that services can be together that depend on one another

File: main-service/src/server.js
Comment (jakublaz) [reply main-service/src/server.js:1]: there were necessary changes in this file so that it will transfer register requests between browser-facade and auth-service

Comment (piotr-ladkowski) [reply main-service/src/server.js:1]: I think we should use branches to paralellize work

The flow of user registration is a big transaction with many services ( browser-facade, main-service, auth-service and active-sessions)

To accomplish this task, I think we should work in paralell on subtasks - for each service.

Currently this code causes conflicts with the code on _piotrek-dev-main-service_ branch, and creates mixup between  development styles.

**Review by piotr-ladkowski — APPROVED** [review]

File: main-service/src/server.js
Comment (jakublaz) [reply main-service/src/server.js:1]: Yes sure, do you know how i can test my app now having server ready to which i send requests?
